### DISCUSSION TITLE
How can i preserve albums as albums?

### DISCUSSION THREAD

**Discussion by AlexRabbit** [aahnik/tgcf#91] [Q&A]
Every time I use this amazing script everything goes well except that albums are forwarded as individual media is there a fix?

**Comment by aahnik** [comment]
well, can you provide a screenshot @AlexRabbit ?

**Reply by AlexRabbit** [reply]
> 
> 
> well, can you provide a screenshot @AlexRabbit ?
yup i can
this is the source channel as you can see its an album

![image](https://user-images.githubusercontent.com/10791909/116318557-92763080-a77a-11eb-9f81-a339f12e584c.png)

this is how the script forwards them

![image](https://user-images.githubusercontent.com/10791909/116318606-a02bb600-a77a-11eb-8293-ca468825387c.png)
|
it forawards them as individual pics.

is there a way to fix that?

**Reply by aahnik** [reply]
@AlexRabbit thanks a lot for the screenshots. I will try to fix this!

**Reply by hndrk-themer** [reply]
Any news about that? Got the same issue on my usecase, as people want to forward 3 Screenshots with an undertitle and a hashtag included to trigger. but just the last screenshot with the trigger gets forwarded but it would be cool to forward the group id if this is existent

**Reply by songxff** [reply]
> @AlexRabbit thanks a lot for the screenshots. I will try to fix this!

Hello, is this problem solved?

**Comment by ne0x86** [comment]
Any update? Still happening :(

**Comment by kotjj** [comment]
Добрый день, проблема очень актуальна, есть решение?

**Comment by panikajo** [comment]
Доброго дня ця проблема і в мене також.

**Comment by chenbb1106** [comment]


**Reply by MobCode100** [reply]
Yes, there is solution to that, but we must wait for the owner to be active again. I might try to fix it on my free time.

**Reply by fedeD84** [reply]
Someone have solution? Album in telethon is in from 2021...

**Reply by quan88** [reply]
> Yes, there is solution to that, but we must wait for the owner to be active again. I might try to fix it on my free time.

Can share your costom code ? Thank so much.

**Comment by quan88** [comment]
Still waiting custom fix code ! Anyone help ? Thank so much.

**Comment by Rafinikus** [comment]
Does anyone have a solution to this problem or not? We've been waiting for 4 years already.
